alluxio/posix/config.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import yaml
+import os
+
+from alluxio.posix.ufs.alluxio import validate_alluxio_config, update_alluxio_config
+from alluxio.posix.ufs.oss import validate_oss_config, update_oss_config
+from alluxio.posix.const import Constants
+from alluxio.posix.exception import ConfigMissingError, ConfigInvalidError
+
+
+class ConfigManager:
+    def __init__(self):
+        self.logger = logging.getLogger(__name__)
+        logging.basicConfig(level=logging.INFO)
+        self.config_file_path = os.getenv('ALLUXIO_PY_CONFIG_FILE_PATH', 'config/ufs_config.yaml')
+        self.config_data = self._load_config()
+        self.validation_functions = {
+            Constants.OSS_FILESYSTEM_TYPE: validate_oss_config,
+            Constants.ALLUXIO_FILESYSTEM_TYPE: validate_alluxio_config
+        }
+
+    def _load_config(self):
+        if not os.path.exists(self.config_file_path):
+            raise FileNotFoundError(f"{self.config_file_path} does not exist.")
```

Comment:
It does not necessarily depend entirely on the configuration file, and can be configured directly through hot update without a configuration file. If there is no corresponding configuration during use, an exception can be thrown directly.